examples/preprocessing/plot_all_scaling.py

```diff
@@ -182,12 +182,13 @@ def plot_distribution(axes, X, y, hist_nbins=50, title="",
 
 
 def make_plot(item_idx):
-    _, X = list(distributions.items())[item_idx]
+    title, X = distributions[item_idx]
     ax_zoom_out, ax_zoom_in, ax_colorbar = create_axes()
     axarr = (ax_zoom_out, ax_zoom_in)
     plot_distribution(axarr[0], X, y, hist_nbins=200,
                       x0_label="Median Income",
-                      x1_label="Number of households")
+                      x1_label="Number of households",
+                      title=title)
```

Comment:
@glemaitre in create_axes you want to use fig.suptitle or something similar to show the title in the middle.